src/lib/execution_plan.rs

```diff
@@ -120,6 +120,26 @@ fn should_skip_workspace_member(member: &str, skipped_members: &HashSet<String>)
     }
 }
 
+fn get_included_workspace_members(include_members_config: String) -> Option<HashSet<String>> {
+    let list = get_skipped_workspace_members(include_members_config);
```

Comment:
this is a bit unreadable to me.
maybe we can have a single function like is_workspace_member_found(member, members_map) -> bool
and then should_skip_workspace_member calls it and if found returns skip and should_include_workspace_member calls it and if found returns include.
same logic should also be for get_skipped_workspace_members and get_included_workspace_members.

Reply:
Done. Did I get it right?